Tools/autotest/rover.py

```diff
@@ -303,6 +303,42 @@ def test_sprayer(self):
         if ex:
             raise ex
 
+    def drive_max_rcin(self, timeout=30):
+        """Test max RC inputs"""
+        self.context_push()
+        ex = None
+
+        try:
+            self.progress("Testing max RC inputs")
+            self.change_mode("MANUAL")
+
+            self.wait_ready_to_arm()
+            self.arm_vehicle()
+
+            self.set_rc(3, 2000)
+            self.set_rc(1, 1000)
+
+            tstart = self.get_sim_time_cached()
```

Comment:
Not a blocker, but I usually make this one a `get_sim_time` as it's nice to get an up-to-date timestamp as you're coming in.  `set_rc` will effectively do that, so it doesn't matter in this case.

Reply:
Done